pyvss/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pyvss/const.py,sha256=aZsaFz1S37-VVZsV8E66taN7pgbD7trfx3mH7iWW23Q,385
pyvss/enums.py,sha256=DoIMJ9YPbvwtmjP91m9Gho7W-NGi7sxWjDh1McvWa6I,1409
pyvss/exceptions.py,sha256=njHriooRRu4eCJtjXEemZ5i3XhajrGd0EBblQ6uk3bA,39
pyvss/helper.py,sha256=fQTZe4rLHj6IH2TDK39xwzioOWf_H0UtxtwiC9vegQc,2534
pyvss/manager.py,sha256=ppmlWTGNUW23I5d_kU4mIUXhB1lhVK--gExDYu_9UZY,185735
pyvss-0.9.41.dist-info/LICENSE,sha256=S-66BixOgVGxq5L07VPMJgSQVMtUbT8OVkJVLtb9hyo,1077
pyvss-0.9.41.dist-info/METADATA,sha256=1vggI802pq0SrrvUMDsPcILJxMx1clh2BhAxJVQEbvI,8609
pyvss-0.9.41.dist-info/WHEEL,sha256=p46_5Uhzqz6AzeSosiOnxK-zmFja1i22CrQCjmYe8ec,92
pyvss-0.9.41.dist-info/top_level.txt,sha256=G2sITBKMrEXH1oNfoVLXQWXi3KE0kw1QbA95o5cb7x4,6
pyvss-0.9.41.dist-info/RECORD,,
